Common/FeatureFlags.swift

```diff
@@ -89,6 +90,13 @@ struct FeatureFlagConfiguration: Decodable {
         #else
         self.includeServicesInSettingsEnabled = true
         #endif
+        
+        // Swift compiler config is inverse, since the default state is enabled.
```

Comment:
@novalegra This line can be deleted since it no longer the inverse.

Reply:
I'll pull it out in my next PR :)